client/src/components/History/GanttVisualisation/Gantt.vue

```diff
@@ -253,4 +357,32 @@ function createClassWithCSS(selector, style) {
 .gantt .tick {
     stroke: #666;
 }
+.gantt {
+    margin-top: 50px;
```

Comment:
Please notice the popup position because of this property, Maybe rather have a margin-bottom to the buttons.